dts/arm/st/wb0/stm32wb0.dtsi

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright (c) 2024 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm/armv6-m.dtsi>
+#include <zephyr/dt-bindings/i2c/i2c.h>
+#include <zephyr/dt-bindings/adc/adc.h>
+#include <zephyr/dt-bindings/pwm/pwm.h>
+#include <zephyr/dt-bindings/gpio/gpio.h>
+#include <zephyr/dt-bindings/clock/stm32wb0_clock.h>
+#include <zephyr/dt-bindings/reset/stm32wb0_reset.h>
+#include <zephyr/dt-bindings/dma/stm32_dma.h>
+#include <freq.h>
+#include <mem.h>
+
+/* Helper macro aligned with STM32_RESET() */
+#define STM32_CLOCK(bus, bit) STM32_CLOCK_BUS_ ##bus (1 << bit)
```

Comment:
Quite nice, but I'd rather like to see being applied uniformly over whole family. So, let's log an issue to be sure it doesn't get missed.

Reply:
Created as #78106